filesystem/Filesystem.php

```diff
@@ -79,6 +79,29 @@ public static function removeFolder($folder, $contentsOnly = false) {
 	}
 
 	/**
+	 * Remove a directory, but only if it is empty.
+	 *
+	 * @param String $folder Absolute folder path
+	 * @param Boolean $recursive Remove contained empty folders before attempting to remove this one
+	 * @return Boolean True on success, false on failure. 
+	 */
+	public static function removeFolderIfEmpty($folder, $recursive = true) {
```

Comment:
Yep... That's SS's inconsistent naming right there. as part of 3 you could rename and deprecate the old functions so they are inline with our conventions?